### PR TITLE
Demo/support bot

### PR DESCRIPTION
**Pull request by MartBakler**
Created a demo usecase for a twitter (or other platforms) support bot, which responds to customer issues and creates support tickets depending on the nature of the issue

### CODE REVIEW

File: examples/twitter_support/main.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+def classify_and_respond(tweet_text: str) -> Response:
+    """
+    Respond to the customer support tweet text empathetically and nicely. 
+    Convey that you care about the issue and if the problem was a direct issue that the support team should fix or a question, the team will respond to it. 
+    """
+
+@monkey.align
+def align_respond():
+    input_tweet_1 = "Laia Johnson: I really like the new shovel but the handle broke after 2 days of use. Can I get a replacement?"
+    assert classify_and_respond(input_tweet_1) == Response(requires_ticket=True, response="Hi, we are sorry to hear that. We will get back to you with a replacement as soon as possible, can you send us your order nr?")
```
Comment (JackHopkins) [inline examples/twitter_support/main.py:35]: Can you lint this please @MartBakler? Ideally creating new lines between the kwargs so we don't line wrap.

File: examples/twitter_support/main.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    align_supportticket()
+
+    input_tweet_1 = "Jack Bell: Bro @Argos why did my order not arrive? I ordered it 2 weeks ago. Horrible service"
+    response = classify_and_respond(input_tweet_1)
+    # requires_ticket=True 
+    # response="Hi Jack, we're really sorry to hear about this. We'll look into it right away and get back to you as soon as possible. Could you please provide us with your order number?"
+    
+    if response.requires_ticket:
+        ticket = create_support_ticket(input_tweet_1)
+        # name='Jack Bell'
```
Comment (JackHopkins) [inline examples/twitter_support/main.py:74]: Can you please clean up the remnant comments?

**Review by JackHopkins — CHANGES_REQUESTED** [review]

File: examples/twitter_support/main.py (hunk 2 above)
Comment (MartBakler) [reply examples/twitter_support/main.py:74]: The remnant comments actually are showing what the output is for that patch function, so they are left deliberately for people to get a quick sense of input-output pairs for new datapoints

File: examples/twitter_support/main.py (hunk 1 above)
Comment (MartBakler) [reply examples/twitter_support/main.py]: Updated

**Review by JackHopkins — APPROVED** [review]

**Comment by MartBakler** [comment]
https://github.com/monkeypatch/monkeypatch.py/issues/40

**Pull request merged**